packages/data-stores/src/woocommerce-eligibility/selectors.ts

```diff
@@ -0,0 +1,102 @@
+import { select } from '@wordpress/data';
+import { get } from 'lodash';
+import { STORE_KEY, statusMapping } from './constants';
+import { TransferEligibilityWarning, TransferEligibilityError, TransferEligibility } from './types';
+import type { State } from './reducer';
+
+export const getState = ( state: State ) => state;
+
+export const getAutomatedTransferEligibility = (
+	state: State,
+	siteId: number | null
+): TransferEligibility | null => {
+	if ( ! state || ! state.transferEligibility ) {
+		return null;
+	}
+
+	if ( ! siteId ) {
+		return null;
+	}
+	return state.transferEligibility[ siteId ];
+};
+
+export const getEligibilityHolds = (
+	state: State,
+	siteId: number | null
+): TransferEligibilityError[] | null => {
+	const transferEligibility = select( STORE_KEY ).getAutomatedTransferEligibility( siteId );
+
+	if ( ! transferEligibility ) {
+		return null;
+	}
+
+	const { errors } = transferEligibility;
+
+	if ( ! errors ) {
+		return null;
+	}
+
+	errors.map( ( { code }: { code: string } ) => {
```

Comment:
The `TransferEligibilityError` already declares what the type of `code` is and TypeScript knows that. You can write just:
```ts
errors.map( ( { code } ) => ... );
```
Alos, the result of the `.map` is not used, so why is it called? Maybe you want to return the result instead of the original `errors`?

Reply:
This was code ported over from elsewhere but I can clean that bit up.